### ISSUE TITLE
Bulk edit projects

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#683]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Bulk editing projects. For now it would be applying labels.

### Description
It would be very nice to be able to bulk edit projects, applying labels to several projects at once.

This is **not** high priority, but something we should look at down the line.

Let's look to the GitHub UI.

![image](https://user-images.githubusercontent.com/7606007/100364789-dd8aec80-2ffe-11eb-9a64-0704098dcb04.png)

### Proposed solution
We need to set selection mode to `multiple` for the project list. This means we need to change how we handle selected project (project to show details for).

![image](https://user-images.githubusercontent.com/7606007/106766393-1d430580-663a-11eb-97f8-4de5a3dfc096.png)
